project/Dependencies.scala

```diff
@@ -129,46 +129,55 @@ object Dependencies {
   }
 
   object Jars {
-    lazy val akkaDependencies: Seq[ModuleID] =
+    lazy val utilsDependencies: Seq[ModuleID] =
       Seq(
-        akka.http          % Compile,
-        akka.httpJson      % Compile,
-        akka.httpJson4s    % Compile,
-        akka.slf4j         % Compile,
-        akka.stream        % Compile,
-        akka.actorTyped    % Compile,
-        akka.httpTestkit   % Test,
-        akka.streamTestkit % Test
+        akka.http                  % Compile,
+        akka.slf4j                 % Compile,
+        akka.httpJson              % Compile,
+        akka.httpJson4s            % Compile,
+        spray.spray                % Compile,
+        typesafe.config            % Compile,
+        cats.core                  % Compile,
+        apacheCommons.text         % Compile,
+        logback.classic            % Compile,
+        atlassian.swaggerValidator % Compile,
+        lightbend.logging          % Compile,
+        akka.httpTestkit           % Test,
+        akka.streamTestkit         % Test
       )
 
     lazy val fileDependencies: Seq[ModuleID] =
       Seq(
         aws.s3               % Compile,
-        aws.sts              % Compile,
-        commons.fileUpload   % Compile,
+        typesafe.config      % Compile,
         jsoup.jsoup          % Compile,
         openhtmltopdf.core   % Compile,
         openhtmltopdf.pdfbox % Compile,
-        openhtmltopdf.slf4j  % Compile,
+        openhtmltopdf.slf4j  % Runtime,
```

Comment:
just curiosity, what requires this change?

Reply:
Nothing REQUIRES this change, it's just that the slf4j dep is not used at compile time but just at Runtime, I simply reduced the scope in whom the dependency is used.